conda_smithy/templates/azure-pipelines-linux.yml.tmpl

```
@@ -11,7 +11,19 @@ jobs:
     fetchDepth: {{ clone_depth }}
 {%- endif %}
   - script: |
-         rm -rf /opt/ghc
+         sudo mkdir -p /opt/empty_dir
+         for d in /opt/hostedtoolcache \
+                  /usr/lib/jvm \
+                  /usr/local/.ghcup \
+                  /usr/local/android \
+                  /usr/local/powershell \
+                  /usr/share/dotnet \
+                  /usr/share/swift \
+                  /opt/ghc \
```

This might make it easier (diff-wise) to add and remove from this list in the future

```suggestion
         for d in \
                  /opt/ghc \
                  /opt/hostedtoolcache \
                  /usr/lib/jvm \
                  /usr/local/.ghcup \
                  /usr/local/android \
                  /usr/local/powershell \
                  /usr/share/dotnet \
                  /usr/share/swift \
```